lib/seek/samples/attribute_type_handlers/seek_sample_multi_attribute_type_handler.rb

```diff
@@ -0,0 +1,60 @@
+module Seek
+  module Samples
+    module AttributeTypeHandlers
+      class SeekSampleMultiAttributeTypeHandler < SeekResourceAttributeTypeHandler
```

Comment:
You could make this a subclass of `SeekSampleAttributeTypeHandler`, then for a lot of methods you can just split the array and call `super` to re-use the existing code to lookup the `Sample` object etc.

e.g.
```ruby
def convert(value)
  value = value.split(',') if value.is_a?(String)
  value.uniq.map { |v| super(v) }
end
```